worlds/animal_well/client.py

```diff
@@ -170,6 +215,60 @@ def on_package(self, cmd: str, args: dict):
         if cmd == "Connected":
             self.slot_data = args.get("slot_data", {})
 
+        try:
+            def parseIds(data):
+                for item in data:
+                    if 'type' in item and 'text' in item:
+                        if item['type'] == 'player_id':
+                            item['text'] = self.player_names[int(item['text'])]
+                        elif item['type'] == 'item_id':
+                            item['text'] = item_id_to_name[int(item['text'])]
+                        elif item['type'] == 'location_id':
+                            item['text'] = location_id_to_name[int(item['text'])]
+
+                return data
+
+            if cmd == "PrintJSON":
+                msgType = args.get("type")
+                if msgType == "Chat":
+                    display_to_client(self, "".join(o['text'] for o in parseIds(args.get("data"))))
+                elif msgType == "Hint":
+                    text = "".join(o['text'] for o in parseIds(args.get("data")))
+                    display_to_client(self, text)
+                elif msgType == "Join":
+                    display_to_client(self, args.get('data')[0]['text'])
+                elif msgType == "CommandResult":
+                    display_to_client(self, args.get('data')[0]['text'])
+                elif msgType == "Tutorial":
+                    pass
```

Comment:
```suggestion
                elif msgType == "Tutorial":
                    pass
                elif msgType == "ItemCheat":
                    display_to_client(self, args.get("data")[0]["text"])
```
When using !getitem from the client, it uses the ItemCheat PrintJSON type.

Reply:
Good catch, added!